scripts/update-versions-for-release

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
```

Comment:
It says update version before a release:

1. when would this script get kicked off?
2. why not do it after release to guarantee version is correct?

Reply:
1. this script would be run manually by the dev as part of release prep, before creating the new tag on Github. 
2. commented above. 
if a release fails, then, at that point, we just fix the issue to succeed the release, imo. 

If there is a better way of doing this, we can definitely make changes.

Reply:
Got it, so the commit would be created prior to the release, but not merged until after?

Thanks a lot for taking this, a lot better than updating these manually. I think one way to guarantee version sync in the future would be:

**.travis.yml: add after_success**
`
    - stage: Deploy
      if: type = push AND tag =~ /^v\d+\.\d+(\.\d+)?(-\S*)?$/ AND env(DOCKERHUB_USERNAME) IS present
      script: make release
      env: RELEASE_ASSETS=true
[TODO] after_success: make update-versions-for-release
`

* Travis would kick this script off
* script checks current versions against the latest release artifacts in GitHub, DockerHub, Helm, etc.
* If there is any mismatch, then create a PR to reflect the latest artifact versions
* Our team would be notified of the PR almost immediately after release, which is still a manual step for us to review but probably safer than having it merge automatically

Reply:
Question is more related to the process than the change itself, so approving. Nice work👍🏼